verify/image_verify_test.go

```diff
@@ -111,7 +111,7 @@ var (
 	testDir string
 )
 
-var _ = BeforeEach(func() {
+var _ = BeforeSuite(func() {
```

Comment:
This is a bit problematic in a sense that all tests would now use the same setup. i.e. same home and such so the tests runs are no longer properly isolated. That would allow something in one test to effect another one.

I am very much in favour of the one VM for all your tests so maybe that is fine for our image repo to do. And I certainly can not recommend anything better to make that work so I guess I am cool with this change overall.

Reply:
I may address that by adding a BeforeEach that does a reset but that is executed only for the the specs in `basic_test.go`.

Reply:
Done.